services/ui-src/src/components/export/ExportedModalOverlayReportSection.tsx

```diff
@@ -114,13 +130,37 @@ export function renderModalOverlayTableBody(
           </Tr>
         );
       });
-    case ReportType.MCPAR:
     case ReportType.NAAAR:
-      throw new Error(
-        `The modal overlay table headers for report type '${reportType}' have not been implemented.`
-      );
+      return entities.map((entity, idx) => {
```

Comment:
retrieve and map standard data to table data within row. Great way to reuse `mapNaaarStandardsData`!